shared/general/persistent-storage.md

```diff
@@ -4,28 +4,28 @@ If you have data or configurations that you would like to persist through applic
 
 #### {{ $names.os.lower }} v2.12.0 and above
 
-Beginning with {{ $names.os.lower }} v2.12.0, persistent storage is handled through [named volumes][multicontainer]. The behavior is much the same as persistent storage on older host OS versions. In fact, for single-container applications, the default `docker-compose.yml` sets up a `resin-data` named volume that links to a `/data` directory in the container. The only difference between this and earlier versions is that accessing this data via the host OS is done at `/var/lib/docker/volumes/<APP ID>_resin-data/_data`, rather than the `/mnt/data/resin-data/<APP ID>` location used with earlier host OS versions.
+Beginning with {{ $names.os.lower }} v2.12.0, persistent storage is handled through [named volumes][multicontainer]. The behavior is much the same as persistent storage on older host OS versions. In fact, for single-container fleets, the default `docker-compose.yml` sets up a `resin-data` named volume that links to a `/data` directory in the container. The only difference between this and earlier versions is that accessing this data via the host OS is done at `/var/lib/docker/volumes/<APP ID>_resin-data/_data`, rather than the `/mnt/data/resin-data/<APP ID>` location used with earlier host OS versions.
 
-Named volumes can be given arbitrary names and can be linked to a directory in one or more containers. As long as every release of the application includes a `docker-compose.yml` and the volume name does not change, the data in the volume will persist across updates.
+Named volumes can be given arbitrary names and can be linked to a directory in one or more containers. As long as every release of the fleet includes a `docker-compose.yml` and the volume name does not change, the data in the volume will persist across updates.
 
 When using named volumes, note that:
 
-- If a device is moved to a new application, the old `/data` folder will be automatically purged.
+- If a device is moved to a new fleet, the old `/data` folder will be automatically purged.
 - During the build process, data added to a container directory that is configured to link to a named volume will be copied to the volume the first time it's created on the device.
 
 #### Using a Supervisor with a version >= v10.0.0
 
-Since balena-supervisor v10.0.0, volumes are no longer automatically removed from disk when references to them are removed from an application's `docker-compose` file. This means that it's no longer possible for data to be lost due to the accidental rename of a volume.
+Since balena-supervisor v10.0.0, volumes are no longer automatically removed from disk when references to them are removed from an fleet's `docker-compose` file. This means that it's no longer possible for data to be lost due to the accidental rename of a volume.
```

Comment:
```suggestion
Since balena-supervisor v10.0.0, volumes are no longer automatically removed from disk when references to them are removed from a fleet's `docker-compose` file. This means that it's no longer possible for data to be lost due to the accidental rename of a volume.
```